src/main/java/core/basesyntax/service/activitieswithfruitservice/impl/ActivitiesStrategyImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.activitieswithfruitservice.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.activitieswithfruitservice.ActivitiesStrategy;
+
+public class ActivitiesStrategyImpl implements ActivitiesStrategy {
+    @Override
+    public void performTransaction(String code, FruitTransaction fruitTransaction) {
+        switch (code) {
+            case "b":
+                new BalanceFruitActivitiesServiceImpl().performTransaction(fruitTransaction);
```

Comment:
do not use switch - create Map and fill it once

Reply:
not fixed. you can't just ignore Mentor's suggestions. if you have any questions or need some help - they should be asked in the platform's chats